### DISCUSSION TITLE
Feature Request: Add API Base configuration to OpenAI embedder

### DISCUSSION THREAD

**Discussion by ifsheldon** [meilisearch/product#759] [Feedback & Feature Proposal]
Please consider adding configuration on API base for OpenAI embedder.

Currently it's hardcoded
https://github.com/meilisearch/meilisearch/blob/809e742253511dae3adab970592f5a76f8c7d182/milli/src/vector/openai.rs#L145

but in enterprise env, we may need to access the service with a different URL or a proxy, which is done by changing the URL base. The official `openai` python library support this.

**Comment by macraig** [comment]
Hello @ifsheldon , thanks for your feedback! Would you be able to add your comment to https://github.com/orgs/meilisearch/discussions/677 ? We're using that discussion to gather all the embedder and hybrid search feedback as we prepare to stabilize the feature.

**Reply by ifsheldon** [reply]
Sure!

**Reply by macraig** [reply]
Thanks! I'll close this one then :)
